packages/xs/ezxenstore.0.1.4/opam

```diff
@@ -12,6 +12,7 @@ depends: [
   "jbuilder" {build}
   "cmdliner"
   "logs"
+  "xenctrl"
```

Comment:
We should also add this to the opam file in the source repo.

Reply:
Sure, after this is merged

Reply:
I'll update the source repo but keep this PR as it is.

Reply:
I have updated the source and moved the tag such that it is part of this PR. I've re-run the Travis check.

Reply:
Sorry, I thought I was reading the change on the main PR, not on `xs-opam`